pallets/ds-maps/src/lib.rs

```diff
@@ -0,0 +1,183 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use frame_support::{
+    codec::{Decode, Encode},
+    decl_error, decl_event, decl_module, decl_storage, dispatch, ensure,
+    weights::{Weight},
+    Parameter,
+};
+use frame_system::ensure_signed;
+use pallet_ds_accounts as accounts;
+use accounts::REGISTRAR_ROLE;
+
+mod default_weight;
+#[cfg(test)]
+mod mock;
+#[cfg(feature = "payment")]
+mod payment;
+#[cfg(test)]
+mod tests;
+
+//Not sure, are those derives required?
```

Comment:
Try to remove and see what happens. If still compiles - probably not necessary.

PS Try to compile in both modes: `cargo build` and `cargo test`, dependencies can be different in these modes.

Reply:
the comment in code does not refer to the information above, but to the structure below it. Anyway, will check.